src/models/user.ts

```diff
@@ -0,0 +1,25 @@
+import * as dynamoose from 'dynamoose';
+import { Document } from 'dynamoose/dist/Document';
+import Semester, { SemesterInterface } from './semester';
+
+export const userSchema = new dynamoose.Schema(
+    {
+        id: String,
+        semesters: {
+            type: Array,
+            schema: Semester,
+        },
+    },
```

Comment:
I know user Id might make more sense in terms of logic, but I think we should consider doing owner exactly as you did in the previous ones... thoughts?

Reply:
oooh i see - so make the primary key owner?